airbyte-db/lib/src/main/java/io/airbyte/db/mongodb/MongoDatabase.java

```diff
@@ -34,15 +35,14 @@
   private static final int BATCH_SIZE = 1000;
 
   private final ConnectionString connectionString;
-  private final String databaseName;
+  private final com.mongodb.client.MongoDatabase database;
```

Comment:
maybe it is better to transfer the qualifier to import?

Reply:
Doesn't work as name of the class is the same :(